src/components/Personalization/createCollect.js

```diff
@@ -24,7 +24,7 @@ export default ({ eventManager, mergeDecisionsMeta }) => {
       };
     }
     if (isNonEmptyArray(decisionsMeta)) {
-      mergeDecisionsMeta(event, decisionsMeta);
+      mergeDecisionsMeta(event, decisionsMeta, EventType.DISPLAY);
     }
 
     event.mergeXdm(data);
```

Comment:
@carterworks There are 2 types of events: `display` and `interact`. The `xdm.eventType` is optional, and customer can set any eventType they want to collect . The logic in Target Edge depends on the event type from `xdm.eventType`.  This was adding some complexity because was forcing alloy or the customer to set a specific event type so that the backend could count impressions/conversions properly. 
This is why we needed to add the propositionEventType at the `_experience.decisioning` level that will control the logic in Target Edge. And leave the customer to set any `xdm.eventType` when sending manually the notifications. 

We will not remove the `xdm.evenType` since it is a field related to the event itself.